src/webapi/xml_http_request.rs

```diff
@@ -91,6 +91,31 @@ impl XMLHttpRequest {
         };
     }
 
+    /// Returns the string containing the text of the specified header. If there
+    /// are multiple response headers with the same name, then their values are
+    /// returned as a single concatenated string.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/getRequestHeader)
```

Comment:
Hmm... that link doesn't lead to anything, and I'm not seeing a method called `getRequestHeader` on [MDN](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest) nor [in the specs](https://xhr.spec.whatwg.org/).

Are you sure this works? If so where does it come from? O_O

Reply:
Ops. Typo there. Should be response header there.